web/docs/setup-vast/tune.md

```diff
@@ -184,6 +183,34 @@ applies to a single field, `suricata.http.http.url`, and has false-positive rate
 of 0.5%. The second rule creates one sketch for all fields of type `addr` that
 has a false-positive rate of 10%.
 
+### Skip dense index creation
+
+Dense indexes can improve export speed. However they will increase the database
+size. VAST provides an option to not create the indexes and save the disk space.
+
+Dense indexes can be disabled for specific fields or types in the configuration
+section `vast.index`
+
+- `targets` a list of types and fully qualified field names to which the index
+  creation rule should be applied.
+- `create-dense-index` Doesn't create dense indexes for given targets if set to
+  false. By default the value is set to true.
+#### Example
+
+```yaml
+vast:
+  index:
+    rules:
+        # don't create dense indexes for suricata.http.http.url fields
+      - targets:
+          - suricata.http.http.url
+        create-dense-index: false
+        # don't create dense indexes for addr type columns
+      - targets:
+          - :addr
+        create-dense-index: false
+```
+
```

Comment:
Please reword this to use active voice and present tense.

Lead with what problem this solves for the user so that it's clear why they'd want to do this in the first place, i.e., the disk and memory usage vs. query performance trade-off.

Reply:
I am not sure if memory usage is higher with the indexers. It can be, but it also may be not noticeable to the user.
Should i test it separately or just ignore the memory usage information?

Reply:
I think we can omit it for now, the disk usage is much more important.